packages/core/src/linker/component_factory.ts

```diff
@@ -16,7 +16,7 @@ import {NgModuleRef} from './ng_module_factory';
 import {ViewRef} from './view_ref';
 
 /**
- * Represents a component created by a `ComponentFactory`.
+ * Represents a component created by the `createComponent` method of ViewContainerRef class or by the depreciated `ComponentFactory` class .
  * Provides access to the component instance and related objects,
  * and provides the means of destroying the instance.
```

Comment:
I think we can just drop the `ComponentFactory` reference, here is a proposal:
```suggestion
 * Represents a reference to an instantiated component.
 * Provides an access to the component instance itself, as well as related objects,
 * and provides the method to destroy the component instance.
```

Reply:
Agree with @AndrewKushnir suggestion.